### PR TITLE
Erosa/question route

### LINKED ISSUE #19: As an Admin, I would like to C.R.U.D Surveys/Questions so that I am able to keep the questions in the database up to date.
As an Admin, I would like to C.R.U.D Surveys/Questions so that I am able to keep the questions in the database up to date.

### **Acceptance criteria**:
- [x] Create Surveys/Questions route
- [x] Read Surveys/Questions route
- [x] Update Surveys/Questions route
- [x] Delete Surveys/Questions route

### PR DESCRIPTION
**Pull request by Danejerosa1111**
Created routes for questions. 

You can currently PUT, POST, DELETE and READ questions from the survey database.

Closes #19

### CODE REVIEW

File: server/routes/question.js [hunk 1]
```diff
@@ -1,5 +1,119 @@
...
+    !req.body.hasOwnProperty("questionAnswers") || 
+    !req.body.hasOwnProperty("questionActive"))
+    {
+        return res.status(400).sent("POST request on /questions formulated incorrectly." +
+        "Body must contain all 6 required fields, questionID, quiestionTitle, questionText, questionType, questionAnswers, questionActive");
+    }
+  try {
+      thisSurvey = await Survey.updateOne(
+        {surveyID: req.params.surveyID},
+        {$push: {questions: req.body}});
```
Comment (HatchedCode) [inline server/routes/question.js:28]: Was this tested and confirmed that it does the intended action? I had an unintended response when I did this for _responses_, so I am just curious. @Danejerosa1111

**Review by HatchedCode — APPROVED** [review]
One comment I left, but other than that it looks good to go.

In the future, just commit and push the source files that have been modified by you such as the question.js file.

File: server/routes/question.js (hunk 1 above)
Comment (marcoarceo) [reply server/routes/question.js:28]: Code wise it looks good, if this is confirmed to work as Osman has mentioned then I will approve of the request.

Comment (HatchedCode) [reply server/routes/question.js:28]: Already discussed about this. Good to go.

**Pull request merged**